src/libraries/System.Composition.AttributedModel/pkg/System.Composition.AttributedModel.pkgproj

```diff
@@ -4,8 +4,23 @@
     <ProjectReference Include="..\src\System.Composition.AttributedModel.csproj">
       <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
-
-    <HarvestIncludePaths Include="lib/netstandard1.0;lib/portable-net45+win8+wp8+wpa81" />
   </ItemGroup>
+  <!-- Remove TFMs that aren't supported by the package intentionally anymore. -->
```

Comment:
Hopefully we can get rid of this after we enabled the package validation.

Reply:
I would expect the future package validation to catch this as well.  It's an intentional breaking change and should be represented with a suppression.

Rather than add such a target to every pkgproj that needs this, what about having an item that declares the set of frameworks you wish to no longer support.  Then have a common target that implements suppressing those?

Reply:
@ericstj before we talked today I didn't know that we will continue protecting this scenario with the new packaging validation SDK. That's why I went for the fast solution that doesn't require an Arcade update as I thought we will just entirely remove this when we switch to csprojs. Based on our recent discussion it seems that we wanna continue protecting that and that I should meanwhile add an ability to exclude such in the existing packaging package. Will do so.

Reply:
Sounds good.  @Anipik here's an interesting use case for suppressions.  I suspect it won't be the only one.  We should learn from this when designing how the new system permits suppressions.